msg/tools/px_generate_uorb_topic_files.py

```diff
@@ -118,6 +118,14 @@ def generate_output_from_file(format_idx, filename, outputdir, templatedir, incl
         msg_context = genmsg.msg_loader.MsgContext.create_default()
         full_type_name = genmsg.gentools.compute_full_type_name(PACKAGE, os.path.basename(filename))
         spec = genmsg.msg_loader.load_msg_from_file(msg_context, filename, full_type_name)
+        field_name_list = []
+        for field in spec.parsed_fields():
+                field_name_list.append(field.name)
+        try:
+                assert 'timestamp' in field_name_list
+        except AssertionError:
+                print("[ERROR] uORB topic files generator:\n\tgenerate_output_from_file:\tNo 'timestamp' field found in " + spec.short_name + " msg definition!")
```

Comment:
Nice error message :+1: 
Can you also ensure that the type is `uint64_t`, our logging infrastructure depends on that?

Reply:
Sure

Reply:
Done!